boskos/resources.json

```diff
@@ -1,5 +1,822 @@
 [
-{"name": "FAKE_PROJECT_1", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_2", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_3", "type": "project", "state": "free"}
+  {
```

Comment:
a) Why are we checking in a state of free? It seems like that will depend on the state of boskos. In other words I think the config format (what things do we want in boskos) to be slightly different from the state format (where is each thing right now?)


b) Would it make more sense to make this config `[ "TYPE": [ "PROJ1", "PROJ2" ]]`, aka:
```
{
  "project": 
    "initial": "dirty",  // so we clean it up before using it
    "names: [
    "ci-kubernetes-e2e-gke-gpu",
    "k8s-jkns-f8n-gce-upg",
    ...
  ],
  "future_type":
  "initial": "who-knows",
  "names": [
    "future-name", "future-other-name", ...
  ]
}
```